site/guides/README.md

```diff
@@ -1,10 +1,11 @@
 # Reference
 
-- [The Rationale behind kpt](guides/rationale.md)
-- [Namespace provisioning CLI](guides/namespace-provisioning-cli.md)
-- [Namespace provisioning UI](guides/namespace-provisioning-ui.md)
+- [The Rationale Behind kpt](guides/rationale.md)
 - [Porch Installation Guide](guides/porch-installation.md)
+- [Porch UI Installation Guide](guides/porch-ui-installation.md)
 - [Porch User Guide](guides/porch-user-guide.md)
+- [Namespace Provisioning CLI](guides/namespace-provisioning-cli.md)
+- [Namespace Provisioning UI](guides/namespace-provisioning-ui.md)
 - [Tenant Onboarding Guide](guides/tenant-onboarding.md)
```

Comment:
Removing this from the navigation was intentional because `namespace provisioning CLI` guide covers this use-case. I kept the old guide around to avoid breaking the old links (may be removing the guide will be the right thing to avoid the confusion).

Reply:
Removed the links to the Tenant Onboarding Guide from both the sidebar and the guides landing page.